docs/examples/language_model/language_model.ipynb

```diff
@@ -407,20 +408,29 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
-      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
-      "100 33.2M  100 33.2M    0     0  1807k      0  0:00:18  0:00:18 --:--:-- 2195k00:30 1326k\n",
-      "./simple-examples/data/ptb.test.txt\n",
-      "./simple-examples/data/ptb.train.txt\n",
-      "./simple-examples/data/ptb.valid.txt\n",
-      "['ptb.test.txt', 'ptb.train.txt', 'ptb.valid.txt']\n"
+      "['sherlockholmes.test.txt', 'sherlockholmes.train.txt', 'sherlockholmes.valid.txt']\n"
      ]
     }
    ],
    "source": [
-    "!./get_ptb_data.sh\n",
-    "ptb_dataset = !ls ptb.*.txt\n",
-    "print(ptb_dataset)"
+    "TRAIN_PATH = \"./sherlockholmes.train.txt\"\n",
+    "VALID_PATH = \"./sherlockholmes.valid.txt\"\n",
+    "TEST_PATH = \"./sherlockholmes.test.txt\"\n",
+    "PREDICT_PATH = \"./tinyshakespeare/input.txt\"\n",
+    "download(\n",
+    "        \"https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/sherlockholmes/sherlockholmes.train.txt\",\n",
```

Comment:
do you have the hash handy? we can use it to check the file integrity (and handle failed/partial downloads)

Reply:
Thanks for pointing me out, maybe we should add checksum with all external(customized) data resources. :)